### PR TITLE
Change namespace to `azure.monitor.opentelemetry`

### PR DESCRIPTION
**Pull request by lzchen**

### CODE REVIEW

File: azure-monitor-opentelemetry/tests/test_distro.py [hunk 1]
```diff
@@ -1,16 +1,18 @@
 from unittest import TestCase
 from unittest.mock import patch
```
Comment (jeremydvoss) [inline azure-monitor-opentelemetry/tests/test_distro.py:2]: Let's move this to a tests/autoinstrumentation folder

File: azure-monitor-opentelemetry/tests/diagnostics/test_status_logger.py [hunk 2]
```diff
@@ -234,31 +234,31 @@ def test_disabled_log_status_failed_initialization(self, mock_getpid):
         check_file_is_empty()
 
     @patch(
-        "azure.monitor.opentelemetry.distro._diagnostics._status_logger._STATUS_LOG_PATH",
+        "azure.monitor.opentelemetry.diagnostics._status_logger._STATUS_LOG_PATH",
```
Comment (jeremydvoss) [inline azure-monitor-opentelemetry/tests/diagnostics/test_status_logger.py:237]: Is the azure sdk style to leave the module folder without _ but put _ in the filenames themselves? OTel, for instance uses _logs as a folder name.

Comment (jeremydvoss) [reply azure-monitor-opentelemetry/tests/diagnostics/test_status_logger.py:237]: Same goes for the autoinstrumentation folder

File: azure-monitor-opentelemetry/samples/logging/custom_properties.py [hunk 3]
```diff
@@ -21,3 +21,5 @@
 
 # Pass custom properties in a dictionary with the extra argument
 logger.debug("DEBUG: Debug with properties", extra={"debug": "true"})
+
+input()
```
Comment (jeremydvoss) [inline azure-monitor-opentelemetry/samples/logging/custom_properties.py:25]: Is there a reason for adding input to all the log samples?

File: azure-monitor-opentelemetry/tests/diagnostics/test_diagnostic_logging.py [hunk 4]
```diff
@@ -68,7 +68,7 @@ def check_file_is_empty():
 
 
 def set_up(
-    is_diagnostics_enabled,
+    isdiagnostics_enabled,
```
Comment (jeremydvoss) [inline azure-monitor-opentelemetry/tests/diagnostics/test_diagnostic_logging.py:71]: typo?

File: azure-monitor-opentelemetry/tests/diagnostics/test_status_logger.py (hunk 2 above)
Comment (lzchen) [reply azure-monitor-opentelemetry/tests/diagnostics/test_status_logger.py:237]: Yes it's following azure-sdk styling.

File: azure-monitor-opentelemetry/samples/logging/custom_properties.py (hunk 3 above)
Comment (lzchen) [reply azure-monitor-opentelemetry/samples/logging/custom_properties.py:25]: It's to add delay to the python app so it doesn't close before exporting.

File: azure-monitor-opentelemetry/tests/diagnostics/test_diagnostic_logging.py (hunk 4 above)
Comment (lzchen) [reply azure-monitor-opentelemetry/tests/diagnostics/test_diagnostic_logging.py:71]: Fixed

**Review by jeremydvoss — APPROVED** [review]

**Pull request merged**